cmd/hydration-controller/main.go

```diff
@@ -82,6 +82,13 @@ func main() {
 		klog.Fatalf("Failed to register OpenCensus views: %v", err)
 	}
 
+	// Set up env var k8s.container.name for OC agent
+	// Only show warning upon failure as Kustomize metrics are not exported to Cloud
+	// Monitoring, container resource type is not a must
+	if err := os.Setenv("OC_RESOURCE_LABELS", "k8s.container.name=\""+reconcilermanager.HydrationController+"\""); err != nil {
+		klog.Warningf("Failed to setup OpenCensus env var k8s.container.name for %s: %v", reconcilermanager.HydrationController, err)
+	}
+
 	// Register the OC Agent exporter
 	oce, err := kmetrics.RegisterOCAgentExporter()
```

Comment:
Why not modify kmetrics.RegisterOCAgentExporter to be the same as metrics.RegisterOCAgentExporter?

Reply:
Done, changed to same style as in metrics pkg.